hack/make-rules/images.sh

```diff
@@ -28,7 +28,7 @@ source hack/tools/setup-go.sh
 IMAGES="${IMAGES:-cmd/archeio}"
 IFS=" " read -r -a images <<< "$IMAGES"
 # overridable registry to use
-KO_DOCKER_REPO="${KO_DOCKER_REPO:-gcr.io/k8s-staging-infra-tools}"
+KO_DOCKER_REPO="${KO_DOCKER_REPO:-us-central1-docker.pkg.dev/k8s-staging-images/infra-tools}"
```

Comment:
We will also need to patch deploying to pull from here (even in prod we pull from staging by digest currently to avoid the circular dependency, probably we should deploy to staging from staging and to prod from the prod ARs in same-region but that's a bigger change ...)

Reply:
(in k8s.io)

Reply:
/hold cancel

ack